rrmngmnt/filesystem.py

```diff
@@ -89,3 +89,39 @@ def create_script(self, content, path):
                 raise errors.CommandExecutionFailure(
                     executor, cmd, rc, err,
                 )
+
+    def wget(self, url, f_dir):
+            """
+            Download file on the host from given url
+
+            :param url: url to file
+            :type url: str
+            :param f_dir: file directory on host
+            :type f_dir: str
+            :return: absolute path to file
+            :rtype: str
+            """
+            rc = None
+            host_executor = self.host.executor()
+            file_path = os.path.join(f_dir, url.split('/')[-1])
+            cmd = ['wget', '-O', file_path, url]
+            with host_executor.session() as vds_session:
+                wget_command = vds_session.command(cmd)
+                with wget_command.execute() as (_, _, stderr):
+                    counter = 0
+                    wait_progress = False
+                    while rc is None:
+                        line = stderr.readline()
+                        if counter == 1000 or not wait_progress:
+                            counter = 0
+                            self.logger.info(line)
```

Comment:
if you think it is necessary them please add progress handler to allow option to track progress, because in my opinion the logger is not the place where to put it ...